Clean playlists (and move unsorted data) - P2

BellezaEmporium opened the pull request:
This is a follow-up to what I was doing over time.

As always, reviews are appreciated.

BellezaEmporium committed:
AU : Delete the mention of backup

BellezaEmporium committed:
AW : Clean a link and add Nos Isla TV

Source : Telearuba Channel 13 and Nos Isla (Called Channel 23) : https://novus.telearuba.aw/watchlive
Verif : https://www.lyngsat-stream.com/tvcountry/Aruba.html

BellezaEmporium committed:
AZ : Add channels

BellezaEmporium committed:
Update az.m3u

BellezaEmporium committed:
Update az.m3u

BellezaEmporium committed:
BE : Clean a link

BellezaEmporium committed:
BF : Add a channel

BellezaEmporium committed:
Update bf.m3u

BellezaEmporium committed:
BG : Removed a link (dupe)

BellezaEmporium committed:
BH : Clean

BellezaEmporium committed:
BS : Add a channel.

BellezaEmporium committed:
Update unsorted.m3u

BellezaEmporium committed:
CH : Changed a link

BellezaEmporium committed:
CL : Clean broken links + Added channels

BellezaEmporium committed:
CM : Update chunks to playlist

BellezaEmporium committed:
Update unsorted.m3u

BellezaEmporium committed:
Update cn.m3u

BellezaEmporium committed:
US : Move HBO Asia and remove HD F&L

BellezaEmporium committed:
Update unsorted.m3u

BellezaEmporium committed:
Update ar.m3u

LaneSh4d0w commented:
Potential answers to some questions : 

HBO Asia is made by HBO, which is located in the US. Since we sort the channels by the country the channel is originated from, that's why HBO Asia is in the US.

HDFashion & Lifestyle isn't an US channel, it's an Ukrainian one : http://hdfashion.tv/contacts

Lots of dupes were found on the unsorted m3u file, which could explain things about why lots of channels are deleted.

Yes, some Chinese channels got their token lost, because it isn't needed for the channel to work (tested and deleted when working without it)

BellezaEmporium committed:
Merge branch 'master' into patch-2

BellezaEmporium committed:
CL : Fix some channel links (reintegrating some)

The Chili server seems to have disappeared, links moved to the US Unlimited server.

BellezaEmporium committed:
Update cl.m3u

LaneSh4d0w commented:
Ready for review.
